packages/zapp/console/src/components/hooks/Entity/constants.ts

```diff
@@ -0,0 +1,37 @@
+import { ResourceType } from 'models/Common/types';
+import { listTasks } from 'models/Task/api';
+import { listWorkflows } from 'models/Workflow/api';
+import { Workflow } from 'models/Workflow/types';
+import { Task } from 'models/Task/types';
+import { PaginatedFetchFn } from '../types';
+
+type EntityStringMap = { [k in ResourceType]: string };
+
+export const entityStrings: EntityStringMap = {
+  [ResourceType.DATASET]: 'dataset',
```

Comment:
This seems to be duplicated in /components/Entities/constants.ts: entityStrings. can you use one or the other?

Reply:
removed this one!